src/FuelClient_TEST.cc

```diff
@@ -538,12 +538,14 @@ TEST_F(FuelClientTest, DownloadModel)
     EXPECT_EQ(ResultType::FETCH_ALREADY_EXISTS, res5.Type());
   }
 
-  // Download model with a dependency specified within its `model.config`
+  // Download model with a dependency specified within its `model.config`.
+  // The dependency points to fuel.gazebosim.org.
```

Comment:
Is it valuable to also test the other way around, i.e. a `ignitionrobotics` URL that depends on a `gazebosim` URL?

Reply:
The reverse also works. I've just added the ability to use both `fuel.ignitionrobotics.org` and 'fuel.gazebosim.org' at the same time. As long as both server configurations are present, then it'll work.